packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/statistics/statistics.controller.js

```diff
@@ -143,6 +148,9 @@ export default class XdslStatisticsCtrl {
    *          - ping
    *          - traffic:upload,
    *          - traffic:download,
+   *          - error:hec,
+   *          - error:fec,
+   *          - error:crc
```

Comment:
Should this documentation update be set to https://github.com/ovh/manager/pull/5280/files#diff-b7e0c1728a80adde3a7f3ae67855eed43a53d7454180f76de4f1e78e52dea5f9R129 ?

Reply:
??

Reply:
@Steffy29 you're adding the details about `error:hec/fec/crc` for the method `getAccessStatistics`, but just under, you're using these flags for the method `getLinesStatistics`, so the documentation should be updated for the method `getLinesStatistics`, not `getAccessStatistics` no?